blog/2023-02-16-do-you-need-to-build-auth/index.md

```diff
@@ -0,0 +1,124 @@
+---
+slug: do-you-need-to-build-auth
+authors: gao
+tags: [auth, dev, authentication, authorization]
+---
+
+# Do you need to build your own auth for apps?
+
+:::note Prologue
+Although article is inspired from personal dev experience and I tried my best to keep things objective, it's still non-negligible that I'm one of the creators of Logto (an auth product).
+
+I would like to point out this first for a better reading experience.
+:::
+
+I’ve seen a lot of developers are asking questions like “should I build my own auth for my app”. While the answer cannot be simple yes or no, I’d like to write an article to breakdown the implementation and demonstrate the pros and cons to help you decide.
```

Comment:
```suggestion
I’ve seen a lot of developers asking questions like “Should I build my own auth for my app?”. Since the answer cannot be a simple "Yes" or "No", I’d like to write an article to breakdown the implementation and demonstrate the pros and cons to help you decide.
```

Reply:
```suggestion
I’ve seen a lot of developers were asking questions like “Should I build my own auth for my app?”. While the answer cannot be a simple "Yes" or "No", I’d like to write an article to breakdown the implementation and demonstrate the pros and cons to help you decide.
```

Reply:
I don't think the `were` is necessary. Not trying to be a gramma police here, but I think `I've seen people doing something` is better than `I've seen people were doing something`

Reply:
cool